.revive.toml

```diff
@@ -24,10 +24,25 @@ warningCode = 0
 [rule.receiver-naming]
 [rule.time-naming]
 [rule.unexported-return]
-#[rule.indent-error-flow]
+[rule.indent-error-flow]
 [rule.errorf]
 [rule.empty-block]
 [rule.superfluous-else]
 [rule.unused-parameter]
 [rule.unreachable-code]
 [rule.redefines-builtin-id]
+
+[rule.atomic]
+[rule.bool-literal-in-expr]
+[rule.constant-logical-expr]
+[rule.unnecessary-stmt]
+[rule.unused-receiver]
+
+[rule.argument-limit]
+  arguments = [7]
```

Comment:
wow, 7 is a lot!

Reply:
It is more than I would like but there are already functions with 7 in the library. At the very least it will prevent us from adding any that have more than 7 in the future. :-)